src/point.rs

```diff
@@ -187,7 +190,7 @@ impl Point {
     /// Perform a multi-exponentiation operation on the passed scalars and points, using the Pipperger algorithm
     pub fn multimult(scalars: Vec<Scalar>, points: Vec<Point>) -> Result<Point, Error> {
         let mut r = Point::new();
-        let n = scalars.len() as u64;
+        let n = scalars.len();
```

Comment:
Why remove the cast?

Reply:
Because a new version of bindgen map C `size_t` type to `usize` Rust type, which is correct :-)